grammar.js

```diff
@@ -109,6 +109,8 @@ module.exports = grammar({
 
     uninterpreted: $ => /(.|\s)*/,
 
+    block_body: $ => $._statements,
```

Comment:
I think this is a good name even though it does not match [parse.y](https://github.com/ruby/ruby/blob/master/parse.y#L1438) which uses  `compstmt` for this.